README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
+
+Install dependencies
+
+```sh
+task install
+```
+
+Run server and asset build in watch mode
+
+```sh
+task dev
```

Comment:
It may be beneficial to provide additional context or instructions regarding the setup, particularly if there are prerequisites or configurations required before running these commands.